packages/evolution-backend/src/api/helpers/downloadCsvFile.ts

```diff
@@ -11,25 +11,33 @@ import papaparse from 'papaparse';
 import moment from 'moment';
 import knex from 'chaire-lib-backend/lib/config/shared/db.config';
 
+// Restricted responses to exclude from the CSV
+export type RestrictedResponse = 'interview_id' | 'user_email' | 'user_username';
+
 /**
  * Download the CSV file with the expected responses.
  * @param {Object} params - The parameters object.
  * @param {Response} params.res - The response object.
  * @param {string} params.fileName - The name of the CSV file to be downloaded.
  * @param {string[]} params.requestedResponses - The requested responses to include in the CSV.
+ * @param {RestrictedResponse[]} [params.restrictedResponses] - The restricted responses to exclude from the CSV.
  * @example
  * // Example requestedResponses:
  * ['_isCompleted', 'access_token', 'end.commentsOnSurvey']
+ * // Example restrictedResponses:
+ * ['interview_id', 'user_email', 'user_username']
  * @returns {Promise<void>} - The CSV content as a response.
  */
 export const downloadCsvFile = async ({
     res,
     fileName,
-    requestedResponses
+    requestedResponses,
+    restrictedResponses
 }: {
     res: Response;
     fileName: string;
     requestedResponses: string[];
+    restrictedResponses?: RestrictedResponse[];
```

Comment:
I would call this `excludeFields` instead, as it is the feature that is really implemented here, ie not show some fields, whether they are 'restricted', ie private, or simply because you don't want them.

Reply:
Yes, good idea. I will do that.